### PR TITLE
Use convert_model in tests

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

Use mo.convert_model instead of converting via onnx in tests.

- Update __version__ to 2.6.0, mo.convert_model contains check `nncf.__version__ >= 2.6.0`
- Add dump fp32 models
- Disable POT backend for `inception_resnet_v2, levit_128, vgg11`  in test_quantize_conformance.py  (120012)
- Update metrics of test_quantize_conformance.py

| Model                                               | 2023.0.1 | 2023.1.0 | Diff    |
|-----------------------------------------------------|------------|------------|---------|
| timm/efficientnet_b0_backend_OLD_TORCH              | 0.74976    | 0.75538    | 0.0056  |
| timm/efficientnet_b0_BC_backend_POT                 | 0.76878    | 0.76872    | -0.0001 |
| timm/levit_128_backend_ONNX                         | 0.73082    | 0.73184    | 0.0010  |
| timm/mobilenetv2_050_backend_POT                    | 0.64234    | 0.64362    | 0.0013  |
| timm/mobilenetv2_050_BC_backend_POT                 | 0.64376    | 0.6431     | -0.0007 |
| timm/mobilenetv3_small_050_backend_POT              | 0.27062    | 0.54104    | 0.2704  |
| timm/swin_base_patch4_window7_224_backend_OLD_TORCH | 0.79586    | 0.81376    | 0.0179  |
| timm/swin_base_patch4_window7_224_backend_TORCH     | 0.78026    | 0.80696    | 0.0267  |

### Reason for changes

New api in openvino to convert PT models

### Related tickets

117885

### CODE REVIEW

File: tests/post_training/test_templates/test_smooth_quant.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def test_get_abs_max_channel_collector(self):
         ),
     )
     # pylint:disable=protected-access
-    def test__get_nodes_to_smooth_data(self, model_cls, references, tmpdir):
+    def test_get_nodes_to_smooth_data(self, model_cls, references, tmpdir):
```
Comment (KodiaqQ) [inline tests/post_training/test_templates/test_smooth_quant.py:171]: The method has the `_get_nodes_to_smooth_data` name. This is why the test has double underlines.

File: tests/post_training/requirements.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ onnx==1.13.1
 onnxruntime==1.14.1
 pytest
 pytest-cov
-openvino-dev==2023.0.1
+openvino-dev==2023.1.0.dev20230811
```
Comment (KodiaqQ) [inline tests/post_training/requirements.txt:10]: Why was it changed? Do we have any proof that this package is not buggy?

**Comment by AlexanderDokuchaev** [comment]
post_training_quantization/149/

File: tests/post_training/requirements.txt (hunk 2 above)
Comment (AlexanderDokuchaev) [reply tests/post_training/requirements.txt:10]: 2023.1.0.dev20230811 have some bug that was fixed on current release branch.
But with 2023.0.1 mo.convert_model does not support quantized models.
Until dump new openvino package it's the best option.

**Comment by AlexanderDokuchaev** [comment]
Update e2e test will be done in another PR.

**Review by KodiaqQ — APPROVED** [review]

**Comment by AlexanderDokuchaev** [comment]
post_training_quantization/152 - with dump fp32 models

**Review by alexsu52 — COMMENTED** [review]
What about PyTorch and TensorFlow backends?

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**